### PR TITLE
Wizard recipe: yyjson-v0.10.0

### PR DESCRIPTION
**Pull request by AlexKlo**
This pull request contains a new build recipe I built using the BinaryBuilder.jl wizard:

* Package name: yyjson
* Version: v0.10.0

@staticfloat please review and merge.

### CODE REVIEW

File: Y/yyjson/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd yyjson/
+sed -i '/#ifndef yyjson_api_inline/i #define yyjson_api_inline yyjson_api' src/yyjson.h
```
[the remark this answers is not in the document]
Comment (AlexKlo) [reply Y/yyjson/build_tarballs.jl:21]: In this case, if I understand correctly, I just need to remove the line with sed?

File: Y/yyjson/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+cmake --install build
+"""
+
+# These are the platforms we will build for by default, unless further
+# platforms are passed in on the command line
+platforms = supported_platforms()
+
+# The products that we will ensure are always built
+products = [
+    LibraryProduct("libyyjson", :нyyjson)
```
[the remark this answers is not in the document]
Comment (AlexKlo) [reply Y/yyjson/build_tarballs.jl:33]: Oh, indeed! It’s just a typo

File: Y/yyjson/build_tarballs.jl (hunk 1 above)
[the remark this answers is not in the document]
Comment (giordano) [reply Y/yyjson/build_tarballs.jl:21]: Yes

**Comment by AlexKlo** [comment]
@giordano what do I need to do next for the JLL library to appear in JuliaBinaryWrappers?

File: Y/yyjson/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+    DirectorySource("./bundled")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir
+for f in ${WORKSPACE}/srcdir/patches/*.patch; do
+    atomic_patch -p1 ${f}
+done
+cd yyjson/
```
Comment (giordano) [inline Y/yyjson/build_tarballs.jl:20]:
```suggestion
cd $WORKSPACE/srcdir/yyjson
for f in ${WORKSPACE}/srcdir/patches/*.patch; do
    atomic_patch -p2 ${f}
done
```

**Pull request merged**